packages/modular-scripts/src/build/index.ts

```diff
@@ -230,8 +230,8 @@ async function buildStandalone(
       modular: targetPackageJson.modular,
       dependencies: targetPackageJson.dependencies,
       bundledDependencies: targetPackageJson.bundledDependencies,
-      module: jsEntryPoint,
-      style: cssEntryPoint,
+      module: paths.publicUrlOrPath + jsEntryPoint,
```

Comment:
I think you'll need to do path.join to resolve the path. If I pass in "./" as the PUBLIC_URL, it would resolve to ".//static/...etc" with a double slash.

Reply:
> I think you'll need to do path.join to resolve the path. If I pass in "./" as the PUBLIC_URL, it would resolve to ".//static/...etc" with a double slash.

done!